src/content/en/2020/seo.md

```diff
@@ -23,3 +23,835 @@ queries: 07_SEO
+  caption="On-Page Robots Directive Use",
+  description="Bar chart showing robots usage. Meta-Robots is 27.70% for desktop and 27.96% for mobile, X-Robots barely registers on the chart with a mere 0.27% of pages on desktop and 0.40% on mobile",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=99993402&format=interactive",
+  sheets_gid="337739550",
+  sql_file="pages_wpt_bodies_by_device.sqll"
+  )
+}}
+
+When analyzing the usage of the meta robots tag in Lighthouse tests, 0.47% of crawlable pages were found to be noindexed. 0.44% of these pages used a noindex directive and did not disallow crawling of the page in the robots.txt.
+
+The combination of disallow within robots.txt and noindex directive in meta robots were found on only 0.03% of pages. While this method offers 'belt and suspenders' redundancy, a page must not be blocked by a robots.txt file in order for an on-page noindex directive to be effective.
+
+Interestingly, rendering changed the meta-robots tag in 0.16% of pages. While there is no inherent issue with using JavaScript to add a meta robots tag to a page or change its content, SEOs should be judicious in execution. If a page loads with a noindex in the robots meta tag before rendering, [search engines won't run the JavaScript](https://developers.google.com/search/docs/guides/javascript-seo-basics#use-meta-robots-tags-carefully) that changes the tag value or index the page.
+
+#### Canonicalization
+
+Canonical tags -as described by [Google here](https://developers.google.com/search/docs/advanced/crawling/consolidate-duplicate-urls)- are used to specify to search engines which is the preferred canonical URL version to index and rank for a page -the one that is considered to be better representative of it- when there are many URLs featuring the same or very similar content. It is important to note that:
+
+- The canonical tag configuration is used along with other signals to select the canonical URL of a page; it is not the only one.
+- Although self-referencing canonical tags are sometimes used, these aren’t a requirement.
+
+[Last year](../2019/seo#canonicalization), it was identified that 48.34% of mobile home pages were using a canonical tag. This year the number of mobile home pages featuring a canonical tag has grown to 53.61%.
+
+{{ figure_markup(
+  image="seo-presence-of-canonical-tag.png",
+  caption="Presence of Canonical Tag",
+  description="Bar chart showing presence of canonical tags. The majority of our analyzed web pages include a canonical tag, the main part being self-referential (47.88% for desktop and 45.31% for mobile). The percentage of canonicalized web pages is higher on mobile (8.45%) than on desktop (4.1%).",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=1777344456format=interactive",
```

Comment:
@antoineeripret this is not displaying properly. Can you check you published the chart properly?

Reply:
I came here to report the same! :-)

Reply:
Ah this was my fault not @antoineeripret !

This fixes it:

```suggestion
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=1777344456&format=interactive",
```